gst/pkg_config.go

```diff
@@ -1,7 +1,7 @@
 package gst
 
 /*
-#cgo pkg-config: gstreamer-1.0 gstreamer-controller-1.0
+#cgo pkg-config: gstreamer-1.0 gstreamer-allocators-1.0 gstreamer-controller-1.0
```

Comment:
Depending on the controller in the main package was a mistake, the different libraries should map to go subpackages.

Move this into an `allocators` subpackage and depend on the main `gst` package please.

Reply:
Do I also have to move `gst_allocator.go` into this subpackage?

Reply:
yes

Reply:
So you'd need to move everything depending on gstreamer-allocators-1.0